app/main.py

```diff
@@ -1,3 +1,45 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: (int, float)) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, distance1: (int, float)) -> Distance:
+        return Distance(self.km + (
+            distance1.km if isinstance(distance1, Distance) else distance1))
+
+    def __iadd__(self, other: (int, float, Distance)) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other: int or float) -> Distance:
```

Comment:
In future use Union[int, float] or int | float as @katevasilenko says
In real projects, I never saw the usage of (int, float)
'int or float' is so bad

Reply:
Check in other places